src/editor/EditorCtrl.ts

```diff
@@ -265,6 +266,9 @@ export default class EditorCtrl {
 
     const refreshBodies = (externalHtml?) => {
       fixedBody = this.editorService.getFixedBody(bodyEditor, externalHtml);
+      $scope.hasOnlyOneSection = $(fixedBody).length === 0 || $(fixedBody).length === 1 ?
+        true :
+        false;
```

Comment:
Is it equivalent to:  `$(fixedBody).length === 0 || $(fixedBody).length === 1` ?

Reply:
Is it the same as:  `$(fixedBody).length < 2` ?